climpred/tests/test_preprocessing.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+import numpy as np
+import pytest
+import xarray as xr
+
+from climpred.constants import CLIMPRED_ENSEMBLE_DIMS
+from climpred.preprocessing.shared import (
+    load_hindcast,
+    rename_SLM_to_climpred_dims,
+    rename_to_climpred_dims,
+    set_integer_time_axis,
+)
+
+on_mistral = False
+try:
+    host = os.environ['HOSTNAME']
+    for node in ['mlogin', 'mistralpp']:
+        if node in host:
+            on_mistral = True
+            from climpred.preprocessing.shared import get_path
+except KeyError:
+    pass
+
+# check for intake_esm to be installed
+try:
+    import intake
+    import intake_esm
+
+    print(intake_esm.__version__)
+    intake_esm_loaded = True
+except ImportError:
+    intake_esm_loaded = False
+
+
+def preprocess_1var(ds, v='global_primary_production'):
+    return ds[v].to_dataset(name=v).squeeze()
+
+
+@pytest.mark.skipif(not on_mistral, reason='requires to be on mistral.dkrz.de')
+@pytest.mark.parametrize(
+    'inits,members',
+    [([1961, 1962, 1963], [3, 4, 5]), (range(1970, 1972), range(1, 3))],
```

Comment:
For consistency can you use range on the first set as well?

Reply:
I wanted to show that range and lists work